app-i18n/man-pages-ja/man-pages-ja-20180315-r1.ebuild

```diff
@@ -1,18 +1,17 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="6"
+EAPI="8"
 GENTOO_MAN_P="portage-${PN}-20060415"
 
-DESCRIPTION="A collection of manual pages translated into Japanese"
+DESCRIPTION="Collection of manual pages translated into Japanese"
 HOMEPAGE="http://linuxjm.osdn.jp/ https://github.com/hattya/portage-man-pages-ja"
```

Comment:
pkgcheck says `https` is available for `http://linuxjm.osdn.jp/`, however the website doesn't load for me (http too). Maybe it's only temporally. If https works, `SRC_URI` might be updated too.

Reply:
Hm going to that page always times me out but `ebuild ... fetch` seems to say it's fine.

Reply:
`ebuild ... fetch` usually always uses gentoo-mirrors for fetching. If you want to force `ebuild` to use the `SRC_URI` links you need to call it like following:
`GENTOO_MIRRORS="" ebuild .. fetch`